### PR TITLE
[DO NOT MERGE] Nemotron: trtllm_mha + mamba extra_buffer — CI debug for release

### PR DESCRIPTION
**Pull request by b8zhong**
Seems to still be failing.

Debug-only PR for the Nemotron-3-Super NVFP4 + MTP NaN (`verify: target model logits`) on `release/v0.5.13`.

Hypothesis: the NaN comes from the **flashinfer full-attention backend**, not the MoE/mamba changes. This swaps full attention to **trtllm_mha** and switches Mamba scheduling to **extra_buffer** (radix cache on).

Changes (both 8-gpu and 4-gpu Nemotron NVFP4 tests):
- drop `--disable-radix-cache`
- `--attention-backend trtllm_mha`
- `--mamba-scheduler-strategy extra_buffer`
- `--page-size 64` (divides `mamba_track_interval=256`; `mamba_cache_chunk_size` auto-derives)

Rationale: trtllm_mha can't use `page_size=1`, and MambaRadixCache `no_buffer` forces `page_size=1`, so trtllm_mha + radix cache requires `extra_buffer`. This is the same proven combo as the GB300 Qwen3.5 MTP test. PDL/gated-RMSNorm and routed-sf are left intact here, so this isolates the attention-backend axis.

Isolation matrix (all off release/v0.5.13):
- #27810 — release as-is (baseline, expected NaN)
- #27812 — revert routed_scaling_factor fusion
- this PR — trtllm_mha full-attn + extra_buffer

**Do not merge.**

---
### CI States

Latest PR Test (Base): :x: [Run #27282994985](https://github.com/sgl-project/sglang/actions/runs/27282994985)
Latest PR Test (Extra): :x: [Run #27282993837](https://github.com/sgl-project/sglang/actions/runs/27282993837)

### CODE REVIEW

**Comment by b8zhong** [comment]
/rerun-test test/registered/8-gpu-models/test_nvidia_nemotron_3_super_nightly.py test/registered/4-gpu-models/test_nvidia_nemotron_3_super_nvfp4.py
